core/common/src/main/java/alluxio/underfs/UnderFileSystemWithLogging.java

```diff
@@ -577,4 +588,25 @@ public UnderFileSystem getUnderFileSystem() {
   public boolean isSeekable() {
     return mUnderFileSystem.isSeekable();
   }
+
+  // TODO(calvin): General tag logic should be in getMetricName
+  private String getQualifiedMetricName(String metricName) {
+    try {
+      if (SecurityUtils.isAuthenticationEnabled() && AuthenticatedClientUser.get() != null) {
+        return Metric.getMetricNameWithTags(metricName, CommonMetrics.TAG_USER,
+            AuthenticatedClientUser.get().getName(), WorkerMetrics.TAG_UFS,
+            mUnderFileSystem.getUnderFSType());
+      } else {
+        return Metric.getMetricNameWithTags(metricName, WorkerMetrics.TAG_UFS,
```

Comment:
Can we also add another tag to denote these metrics are for UFS operations?

Reply:
We already have a tag for the ufs. Do you mean add a prefix to the metric name, for example `UFSExists` instead of `Exists`?

Reply:
Nevermind, I'll take care of this in another PR.